doc/nrf/protocols/matter/getting_started/hw_requirements.rst

```diff
@@ -602,6 +626,111 @@ For example, for the ``nrf7002dk/nrf5340/cpuapp`` board target, the command is a
 
    west build -b nrf7002dk/nrf5340/cpuapp -t partition_manager_report
 
-.. note::
+Diagnostic logs RAM memory requirements
+=======================================
+
+:ref:`Diagnostic logs support<matter_configuration_diagnostic_logs>` requires the new configuration of the RAM memory to add three new retained partitions.
+The :ref:`matter_diagnostic_logs_snippet` adds the new RAM partitions which are configured as retained to keep the log data persistent and survive the device reboot.
+They also reduce the SRAM size of the Application core according to the size of the retained partition.
+You can adjust the retained partitions for your needs by editing the :ref:`matter_diagnostic_logs_snippet` snippet devicetree file for relevant board.
+
+The following tabs shows the RAM memory division defined for the default configuration of the :ref:`matter_diagnostic_logs_snippet` for each development kits.
```

Comment:
```suggestion
The following tabs shows the RAM memory layout defined for the default configuration of the :ref:`matter_diagnostic_logs_snippet` for each development kit.
```

Reply:
There are multiple additional usages of the term "division" in this doc. I prefer every instance be changed to layout as it explains the birdseye view. Division is a trait internal to the layout